pages/index.js

```diff
@@ -1,7 +1,29 @@
+import ActivityCard from "@/components/ActivityCard";
+import Header from "@/components/Header";
+import { dummyData } from "@/lib/dummyData";
+import styled from "styled-components";
+
 export default function HomePage() {
   return (
-    <div>
-      <h1>Hello from Next.js</h1>
-    </div>
+    <>
+      <Header></Header>
```

Comment:
If your component does not wrap any content, it is common practice to use a self closing tag:
```js
<Header />